css/CSS2/generated-content/content-024-ref.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Reference</title>
+<style>
+    div {
+        border: 2px solid black;
+    }
+</style>
+<body>
+    <p>Test passes if there are squares "&#x25A0;" and "&#x25A0;.&#x25A0;" in the box below.</p>
+    <div>&#x25A0;<br>&#x25A0;.&#x25A0;</div>
```

Comment:
As above, though given the square.square this might be harder/impossible? I guess we might just actually be able to test it using two-arg `counter()`, though I'm not sure the kerning will actually match?

Reply:
No, the kerning doesn't match in Chrome, seemingly. :(